Specification/3._Data_structure_description_language.md

```diff
@@ -583,8 +584,8 @@ There is no difference between the following two array definitions other than te
 
 Both forms are equivalent in terms of the data serialization.
 
-Array items can be of variable length; for example, if the item's type is a data structure that contains dynamic
-arrays itself, so the minimum and maximum size of an array item are not always the same.
+Array items can be of variable length - for example, if the item's type is a data structure that contains dynamic
```

Comment:
This sentence might be better written. *Perhaps* " Array items can be of variable length (for example, if the item's type is a data structure that itself contains dynamic arrays). As a result, array maximum and minimum size may not not always be the same."

In any case "... that contains dynamic arrays itself ..." reads slightly better as "that itself contains dynamic arrays".